osquery/core/utils.h

```diff
@@ -26,4 +30,9 @@ std::string platformStrerr(int errnum);
 
 /// Copies src string into the dst string buffer with error checks
 Status platformStrncpy(char* dst, size_t nelms, const char* src, size_t count);
-}
\ No newline at end of file
+
+/// Takes a Windows FILETIME object and returns seconds since epoch
+#ifdef WIN32
+LONGLONG filetimeToUnixtime(FILETIME ft);
```

Comment:
Why not move this into the platform-build specific `fileops` file? Then you won't need the `ifdef`.